src/main/java/org/gridsuite/study/server/dto/ComputationType.java

```diff
@@ -1,16 +1,60 @@
 package org.gridsuite.study.server.dto;
 
+import lombok.Getter;
+import org.gridsuite.study.server.notification.NotificationService;
+import org.gridsuite.study.server.service.shortcircuit.ShortcircuitAnalysisType;
+
+import static org.gridsuite.study.server.service.shortcircuit.ShortcircuitAnalysisType.ONE_BUS;
+
+@Getter
 public enum ComputationType {
-    LOAD_FLOW("LoadFlow"),
-    SECURITY_ANALYSIS("SecurityAnalysis"),
-    SENSITIVITY_ANALYSIS("SensitivityAnalysis"),
-    SHORT_CIRCUIT("ShortCircuitAnalysis"),
-    VOLTAGE_INITIALIZATION(""),
-    DYNAMIC_SIMULATION("");
+    LOAD_FLOW("LoadFlow", "Load flow", "loadFlowResultUuid",
+            NotificationService.UPDATE_TYPE_LOADFLOW_STATUS, NotificationService.UPDATE_TYPE_LOADFLOW_RESULT,
+            NotificationService.UPDATE_TYPE_LOADFLOW_FAILED),
+    SECURITY_ANALYSIS("SecurityAnalysis", "Security analysis", "securityAnalysisResultUuid",
+            NotificationService.UPDATE_TYPE_SECURITY_ANALYSIS_STATUS, NotificationService.UPDATE_TYPE_SECURITY_ANALYSIS_RESULT,
+            NotificationService.UPDATE_TYPE_SECURITY_ANALYSIS_FAILED),
+    SENSITIVITY_ANALYSIS("SensitivityAnalysis", "Sensitivity analysis", "sensitivityAnalysisResultUuid",
+            NotificationService.UPDATE_TYPE_SENSITIVITY_ANALYSIS_STATUS, NotificationService.UPDATE_TYPE_SENSITIVITY_ANALYSIS_RESULT,
+            NotificationService.UPDATE_TYPE_SENSITIVITY_ANALYSIS_FAILED),
+    SHORT_CIRCUIT("ShortCircuitAnalysis", "Short circuit analysis", "shortCircuitAnalysisResultUuid",
+            NotificationService.UPDATE_TYPE_ONE_BUS_SHORT_CIRCUIT_STATUS, NotificationService.UPDATE_TYPE_ONE_BUS_SHORT_CIRCUIT_RESULT,
+            NotificationService.UPDATE_TYPE_ONE_BUS_SHORT_CIRCUIT_FAILED, ONE_BUS),
+    VOLTAGE_INITIALIZATION("", "Voltage init", "voltageInitResultUuid",
+            NotificationService.UPDATE_TYPE_VOLTAGE_INIT_STATUS, NotificationService.UPDATE_TYPE_VOLTAGE_INIT_RESULT,
+            NotificationService.UPDATE_TYPE_VOLTAGE_INIT_FAILED),
+    DYNAMIC_SIMULATION("", "Dynamic simulation", "dynamicSimulationResultUuid",
+            NotificationService.UPDATE_TYPE_DYNAMIC_SIMULATION_STATUS, NotificationService.UPDATE_TYPE_DYNAMIC_SIMULATION_RESULT,
+            NotificationService.UPDATE_TYPE_DYNAMIC_SIMULATION_FAILED);
 
     public final String subReporterKey;
+    private final String label; // used for logs
+    private String resultUuidLabel;
+    private ShortcircuitAnalysisType oneBus; // only useful for SHORT_CIRCUIT
+    private String updateStatusType;
+    private String updateResultType;
+    private String updateFailedType;
+
+    public void setBusMode(ShortcircuitAnalysisType oneBus) {
```

Comment:
Yep and it was not a good idea to update an enum value. 

I added one more value to the enum (SHORT_CIRCUIT_ONE_BUS). Not perfect because it turns an enum value into two but everyhing is much clearer.

What do you think ? How would you write your subtype suggestion ?